OmgvaPOS/src/ScheduleManagement/Mappers/EmployeeScheduleMapper.cs

```diff
@@ -0,0 +1,57 @@
+﻿using OmgvaPOS.ReservationManagement.Models;
+using OmgvaPOS.ScheduleManagement.DTOs;
+using OmgvaPOS.ScheduleManagement.Models;
+
+namespace OmgvaPOS.ScheduleManagement.Mappers
+{
+    public static class EmployeeScheduleMapper
+    {
+        public static Timeslot ToUnavailableTimeslot (Reservation reservation, TimeSpan duration)
```

Comment:
Not neccesarily in this PR, but in the future, could you use extension methods for mappers?

Reply:
Noted!